src/team_data.rs

```diff
@@ -1,44 +1,67 @@
-use crate::github::GithubClient;
 use anyhow::Context as _;
+use reqwest::Client;
 use rust_team_data::v1::{People, Teams, ZulipMapping, BASE_URL};
 use serde::de::DeserializeOwned;
 
-async fn by_url<T: DeserializeOwned>(client: &GithubClient, path: &str) -> anyhow::Result<T> {
-    let base = std::env::var("TEAMS_API_URL").unwrap_or(BASE_URL.to_string());
-    let url = format!("{}{}", base, path);
+#[derive(Clone)]
+pub struct TeamApiClient {
```

Comment:
Wouldn't it be more consistent to use `TeamClient` instead of `TeamApiGithub`?

After all, we don't have a `GithubApiClient` but just a `GithubClient` despite it being `api.github.com`, probably similar for `ZulipClient`.

Reply:
Fair enough! Done.